source/_posts/2022-05-16-home-assistant-os-release-8.markdown

```diff
@@ -0,0 +1,120 @@
+  - [Under the Hood](#under-the-hood)
+  - [Other Changes](#other-changes)
+- [Device Support](#device-support)
+  - [Raspberry Pi](#raspberry-pi)
+  - [Home Assistant Yellow](#home-assistant-yellow)
+  - [Generic x86-64](#generic-x86-64)
+  - [Generic AArch64 support](#generic-aarch64-support)
+
+## Operating System Changes
+
+### GRUB2 for UEFI based systems
+
+For Generic x86-64, OVA and the new Generic AArch64 Home Assistant uses GRUB2
+as boot loader now. GRUB2 (GRand Unified Bootloader) is the de-facto standard
+boot loader used by most Linux distributions. The main reason switching from
+Barebox to GRUB2 was the missing AArch64 UEFI boot support in Barebox. We also
+expect GRUB2 to be more stable especially on Desktop style x86-64 systems as
+it gets used by much more users since generic Linux distributions use GRUB2.
+Although, we actually hit a bug in GRUB2 during the RC phase, but let's hope
+this was a one-off. 🤞
+
+<img src='/images/blog/2022-05-16-os8/haos-grub2-menu.png' alt='Screenshot showing GRUB2 menu of Home Assistant OS'>
+
+The boot menu shows the two boot slot. Typically you don't have to change
+selection here, unless you intentionally want to boot the previously installed
+Home Assistant OS version.
+
+Note: Upgrading from any version 7.x is safe, but we recommend to upgrade from
+the last version of the previous major release. This is also the best tested
+upgrade path. From any release with GRUB2 it is only safe to downgrade to 7.6!
+Downgrading to releases before 7.6 can be done by downgrading to 7.6 first.
+
+### Under the Hood
+
+Under the hood, the OS was updated to the latest upstream Linux 5.15 kernel
+as well as Buildroot 2022.02.1. The latest Buildroot release brings new
+versions of various core components like systemd 250, NetworkManager 1.34.0
+and Docker 20.10.14.
+
+Additional networking drivers and settings prepare Home Assistant OS to
+host the [OpenThread Border Router add-on].
+
+### Other Changes
+
+- IP set support for advanced firewalling (also used by the OTBR add-on)
+- Support for NTP configuration via DHCP
+- Google Coral support is now using Google's latest driver. This enables
+  additional Coral device support such as PCI Dual Edge TPU.
+- Legacy wext backend for wpa_suppilcant is now enabled to support more WiFi
+  devices.
+
+## Device Support
+
+### Raspberry Pi
+
+All Raspberry Pi versions use the latest LTS Linux Kernel 5.15 and firmware
+(tag 1.20220331) from the Raspberry Pi team. These are the same versions as
+the Raspberry Pi OS is using currently.
+
+### Home Assistant Yellow
+
+This is the first release which supports Home Assistant Yellow. Since Home
+Assistant Yellow uses the Raspberry Pi Compute Module 4, the support is
+based of the regular Raspberry Pi support currently. The Yellow image is
+using the same kernel and firmware version. The Yellow board also supports
+booting directly off of a NVMe device for those using a CM4 Lite (without
```

Comment:
```suggestion
booting directly off of an NVMe device for those using a CM4 Lite (without
```

Reply:
`An` is only used before vowels not? NVMe starts with a `n`.

Reply:
It is not just about the first letter, it's about the sound they make; if it sounds like a vowel, we still use the "an" article. Especially with abbreviations, this is a thing (NVMe `en-fee-em-eee`)

Reply:
Today I learned :)